packages/core/src/connectors/types.ts

```diff
@@ -1,6 +1,11 @@
 import { Languages } from '@logto/phrases';
-import { ConnectorConfig, ConnectorType } from '@logto/schemas';
+import { ConnectorConfig, Connector } from '@logto/schemas';
 
+export enum ConnectorType {
```

Comment:
duplicate `db-entries/custom-types.ts`

Reply:
`db-entries/custom-types.ts ` this file is auto-generated, as we removed `type` from connectors schema, `ConnectorType` no longer exists in this file.

Reply:
`db-entries/custom-types.ts ` this file is auto-generated, as we removed `type` from connectors schema, `ConnectorType` no longer exists in this file.